tests/playwright/src/specs/container-smoke.spec.ts

```diff
@@ -113,7 +113,14 @@ describe('Verification of container creation workflow', async () => {
     images = await navigationBar.openImages();
     playExpect(await images.getCurrentStatusOfImage(imageToPull)).toBe('USED');
   });
-
+  test('Test navigation between pages', async () => {
+    const navigationBar = new NavigationBar(page);
+    const containers = await navigationBar.openContainers();
+    const containersDetails = await containers.openContainersDetails(containerToRun);
+    await playExpect(containersDetails.heading).toBeVisible();
+    await containersDetails.backLink.click();
+    await playExpect(containers.heading).toBeVisible();
```

Comment:
Could you please also add a Close Details button test? as a second part of navigation test?